lib/workers/repository/update/pr/changelog/release-notes.spec.ts

```diff
@@ -97,11 +97,12 @@ describe('workers/repository/update/pr/changelog/release-notes', () => {
     it('returns input if invalid', async () => {
       const input = { a: 1 };
       expect(
-        await addReleaseNotes(input as never, {} as BranchUpgradeConfig)
+        await addReleaseNotes(input as never, partial<BranchUpgradeConfig>({}))
```

Comment:
we should allow partial function call without empty object, like `partial<Type>()`.

Reply:
https://github.com/renovatebot/renovate/pull/20618